creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -172,477 +172,476 @@ const handleSyncFromPrimary = async ({
     )
 
```

Comment:
ouch this diff lol

Reply:
ahh its primarily just removing the `for (const fetchedCNodeUser of Object.values(body.data.cnodeUsers)) {` right?

Reply:
yeah mind summarizing what changes you made here @theoilie?

Reply:
ok diff makes much more sense once i removed the for loop from master and then compared
https://www.diffchecker.com/lu7RTJAa